packages/postgres-database/src/simcore_postgres_database/migration/versions/d4a26cf4e9cb_add_user_preferences_table.py

```diff
@@ -0,0 +1,40 @@
+"""add user preferences table
+
+Revision ID: d4a26cf4e9cb
+Revises: e987caaec81b
+Create Date: 2023-08-24 09:48:19.726664+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "d4a26cf4e9cb"
+down_revision = "e987caaec81b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "user_preferences",
```

Comment:
I think user preferences should be on the user + product level, so I would add a product column there.

Reply:
OK, so I have changed this one and also encoded the product_name inside the `user_preference_name` column. This will be requested int the UserPreferenceRepo utils that you are suppose to use when saving/loading them